src/components/interactive-map/interactive-map.tsx

```diff
@@ -292,7 +324,7 @@ const InteractiveMap: React.FC<InteractiveMapProps> = (props) => {
                         y={y}
                         width={ORDER_FAILED_CROSS_WIDTH}
                         length={ORDER_FAILED_CROSS_LENGTH}
-                        angle={angle}
+                        angle={45}
```

Comment:
Yeah, I saw your comment about this. So the `angle` value that gets passed to this callback is the angle of the arrow being drawn on, so my thinking was we would either do `angle`  or `angle + 45`  or something. But if what you have works, that's fine.

Reply:
Let's leave it at this then.